app/services/idv/steps/send_link_step.rb

```diff
@@ -8,9 +8,14 @@ class SendLinkStep < DocAuthBaseStep
       def call
         return throttled_failure if throttle.throttled_else_increment?
         telephony_result = send_link
+        failure_reason = nil
+        unless telephony_result.success?
```

Comment:
my personal preference is to avoid `unless` because of the mental gymnastics
```suggestion
        if !telephony_result.success?
```